### PR TITLE
aw - wfm info

### PR DESCRIPTION
**Pull request by aw-west-defra**
I have previously had a dataset "business_info" which calculated arable/grassland which are commonly asked for in reactive analysis.
_(There was also a few other metrics, I've forgotten them, but they can be added later)_

I wanted to formally save it, and I feel this can be merged with centroids from issue #114, and branch [114-parcel-centroids-for-elmo](https://github.com/Defra-Data-Science-Centre-of-Excellence/elmo-geo/tree/114-parcel-centroids-for-elmo)

### CODE REVIEW

File: notebooks/ingestion/wfm_info.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    "ha_unimproved",
+    "ha_unimproved_disadvantaged",
+    "ha_disadvantaged",
+    "ha_severely_disadvantaged",
+    "ha_moorland",
+    "ha_fenland",
+]
+
+
+sdf = (
```
Comment (obi-defra) [inline notebooks/ingestion/wfm_info.py:65]: sdf has duplicate id_parcel entreis due to parcel geometries being split over sindex tiles.

I don't think the silver dataset should contain these duplicates, as the only columns that differ are the x and y coords. This also means there is a separate centroid for each part of the parcel.

So needs a groupby + ST_Union_Aggr before the centroid calculation.

**Review by obi-defra — CHANGES_REQUESTED** [review]
Splitting or parcel geometries results in multiple centroid coordinates per id_parcel. Requires a groupby operation in there somewhere.

File: notebooks/ingestion/wfm_info.py (hunk 1 above)
Comment (aw-west-defra) [reply notebooks/ingestion/wfm_info.py:65]: I believe I've fixed this.
I also added asserts to check it's true.

**Review by obi-defra — APPROVED** [review]
Changes look good.

Check by loading field and farm info data frames, checking for duplicates and checking coordinates not null.

**Pull request merged**